### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by KoushikKintali**
landing-page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    margin: 8rem 8rem;
+    height: 100%;
+}
+
+.landing-page-content {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    /* height: 100vh; */
```
Comment (msdeshmukh009) [inline index.css]: You can remove this comment if you are not planning to use it.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.section-tile {
+    margin: 1.5rem;
+}
+
+
```
Comment (msdeshmukh009) [inline index.css]: Extra space over here.

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking nice. Good to go!!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>neoBooks</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined">
+    <link rel="stylesheet" href="./index.css"></link>
+    <link rel="stylesheet" href="./main.css"></link>
+</head>
+<body>
+    <nav class="nav">
+        <a class="nav-logo" src="" alt="logo">
```
Comment (Kothalkarpriya) [inline index.html]: src is empty if not needed do remove

File: index.html [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+        <section class="landing-page-content">
+            <aside class="side-content">
+                <p class="side-content-header">India's Largest</p>
+                <p class="side-content-header">Online neoBookstore</p>
+                <p class="side-content-description">Book your Books and get it delivered at your doorstep</p>
+                <div class="flex-container flex-row">
+                    <button onclick="location.href = './components/alert/alert.html'" href="./components/alert/alert.html" class="btn btn-lg btn-fill">Order Now</button>
+                </div>
+            </aside>
+            <img class="side-image" src="./assets/landing-page.png" alt="man-with-book">
```
Comment (Kothalkarpriya) [inline index.html]: IMG tag is self-closing so do use / before closing it

File: index.html [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+        </section>
+        <section class="section-body">
+            <div class="section-title category">Categories</div>
+            <div class="section-group-tiles">
+                <div class="card section-tile category">
+                    <div class="card-body category-body">
+                        <div class="card-img">
+                            <img class="category-img" src="./assets/fantasy.png">
```
Comment (Kothalkarpriya) [inline index.html]: every time use use img tag do make the changes

**Review by Kothalkarpriya — APPROVED** [review]
do make the suggested changes and then you can merge

**Review by manojasarsa — APPROVED** [review]
The code looks clean. Good to merge!

**Pull request merged**